pkg/controller/direct/maputils.go

```diff
@@ -26,6 +26,7 @@ import (
 	"google.golang.org/protobuf/reflect/protoreflect"
 	"google.golang.org/protobuf/types/known/durationpb"
 	"google.golang.org/protobuf/types/known/timestamppb"
+	"google.golang.org/protobuf/types/known/wrapperspb"
```

Comment:
So this library is already deprecated, I was not sure if we should support it or not. Now it is more obvious that we do want to support it.  Could you add the other wrapperspb  functions in this lib or improve the mapper generator to consider the types from wrapperspb (as a bonus)?